shared/src/main/scala/scala/xml/PrettyPrinter.scala

```diff
@@ -23,8 +23,11 @@ import Utility.sbToString
  *  @param width the width to fit the output into
  *  @param step  indentation
  */
-class PrettyPrinter(width: Int, step: Int) {
+class PrettyPrinter(width: Int, step: Int, minimizeEmpty: Boolean) {
```

Comment:
Could you please document the expected behaviour when this parameter is set to true? (e.g. in `@param`)